src/util/measurementUncertaintyTool.js

```diff
@@ -0,0 +1,66 @@
+import Decimal from 'decimal.js';
+
+// Run node version with 14.18.3
+/**
+ * Custom measurement uncertainty rounding rules.
+ * @param  {number} inputValue            measured value from annotation
+ * @param  {number} uncertaintyValue      uncertainty from pixel diagonal
+ */
+
+/**
+ * Rounding
+ */
+
+function roundValue(inputValue, uncertaintyValue) {
```

Comment:
I think it is important to keep the function docu immediately over it
```suggestion
/**
 * Custom measurement uncertainty rounding rules.
 * @param  {number} inputValue            measured value from annotation
 * @param  {number} uncertaintyValue      uncertainty from pixel diagonal
 */
function roundValue(inputValue, uncertaintyValue) {
```

Reply:
I think that this function can get some of the logic extracted to smaller functions so it is easier to test and read